DependencyInjection/CmfRoutingExtension.php

```diff
@@ -75,102 +79,81 @@ public function setupFormTypes(array $config, ContainerBuilder $container, Loade
      */
     private function setupDynamicRouter(array $config, ContainerBuilder $container, LoaderInterface $loader)
     {
+        $loader->load('routing-dynamic.xml');
+
         // strip whitespace (XML support)
         foreach (array('controllers_by_type', 'controllers_by_class', 'templates_by_class', 'route_filters_by_id') as $option) {
-            $config[$option] = array_map(function ($value) {
-                return trim($value);
-            }, $config[$option]);
+            $config[$option] = array_map('trim', $config[$option]);
         }
 
         $defaultController = $config['default_controller'];
         if (null === $defaultController) {
             $defaultController = $config['generic_controller'];
         }
-        $container->setParameter($this->getAlias() . '.default_controller', $defaultController);
-        $container->setParameter($this->getAlias() . '.generic_controller', $config['generic_controller']);
-        $container->setParameter($this->getAlias() . '.controllers_by_type', $config['controllers_by_type']);
-        $container->setParameter($this->getAlias() . '.controllers_by_class', $config['controllers_by_class']);
-        $container->setParameter($this->getAlias() . '.templates_by_class', $config['templates_by_class']);
-        $container->setParameter($this->getAlias() . '.uri_filter_regexp', $config['uri_filter_regexp']);
-        $container->setParameter($this->getAlias() . '.route_collection_limit', $config['route_collection_limit']);
-
-        $locales = empty($config['locales']) ? array() : $config['locales'];
-        $container->setParameter($this->getAlias() . '.dynamic.locales', $locales);
-        if (count($locales) === 0 && $config['auto_locale_pattern']) {
+        $container->setParameter('cmf_routing.default_controller', $defaultController);
+
+        $locales = $config['locales'];
+        if (0 === count($locales) && $config['auto_locale_pattern']) {
```

Comment:
to me that does not need to be a fix rule. the important is that things that logically belong together should visually stick together. and i would not do the assignment if we only use the value a few times as argument.